src/store/modules/ConnectModule.js

```diff
@@ -25,6 +25,10 @@ class ConnectModule extends VuexModule {
     return clientId;
   }
 
+  get server() {
```

Comment:
don't need to get `.server`, 
you can just check window.location for check current state